src/client/hud.cpp

```diff
@@ -522,8 +522,8 @@ void Hud::drawLuaElements(const v3s16 &camera_offset)
 				client->getMinimap()->drawMinimap(rect);
 				break; }
 			default:
-				infostream << "Hud::drawLuaElements: ignoring drawform " << e->type <<
-					" of hud element ID " << i << " due to unrecognized type" << std::endl;
+				infostream << "Hud::drawLuaElements: ignoring drawform " << e->type
```

Comment:
The server HUD ID might still be helpful here to figure out which HUD is affected.

Reply:
The small reorganization up top unfortunately makes this impossible.